ccloud/ccloud-generate-cp-configs.sh

```diff
@@ -59,20 +59,30 @@ set -eu
 
 ################################################################################
 # Confluent Cloud configuration
+#
+# Example file at $HOME/.ccloud/config
+#
+#   $ cat $HOME/.ccloud/config
+#   bootstrap.servers=<BROKER ENDPOINT>
+#   ssl.endpoint.identification.algorithm=https
+#   security.protocol=SASL_SSL
+#   sasl.mechanism=PLAIN
+#   sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username\="<API KEY>" password\="<API SECRET>";
+#
 ################################################################################
 CCLOUD_CONFIG=$HOME/.ccloud/config
```

Comment:
I think it might be preferrable to make this ENV_VAR consistent across the scripts.   `start.sh`, for example, uses `CONFIG_FILE`, where this uses `CCLOUD_CONFIG`.  I thought we might want to make overriding the variables easier in a future PR, so maybe we could align the env var names now as a precursor to that?

Reply:
@rspurgeon good catch and agree on consistency. For this PR, I will standardize on `CONFIG_FILE` (it's easier to change just one file).  Addressed in commit https://github.com/confluentinc/examples/pull/208/commits/bb5ec1f90fc6dd1a774f68b82c2e881634587543